OneCommunityGlobal/HGNRest
# Ivy-reasonSche-unit-test

# Description
Please include the exact bug/functionality description and a summary of the changes/ related issues. Please also include any other relevant motivation and context:
Fixes # (bug list priority high/medium/low x.y.z)
Or Implements # (WBS) 

## Related PRS (if any):
This frontend PR is related to the #XXX backend PR.
To test this backend PR you need to checkout the #XXX frontend PR.
…

## Main changes explained:
- Delete file A for removing unused components …
- Update file B for including new pattern …
- Create file C for introducing new components …
…

## How to test:
1. check into current branch
2. do `npm install` and `npm run build` to run this PR locally
3. do `npm run test reason `
<img width="731" alt="Screenshot 2024-08-29 at 4 35 01 PM" src="https://github.com/user-attachments/assets/72cbabc0-3d09-491f-9075-495fa4e8caae">


## Screenshots or videos of changes:

## Note:
Include the information the reviewers need to know.
